### PR TITLE
Bug 1032371 - Display users who have completed task on task detail page

### PR DESCRIPTION
**Pull request by tessie**

### CODE REVIEW

**Comment by tessie** [comment]
@bobsilverberg .reopening to fix issue sort users are listed  in the order most recently completed first.https://bugzilla.mozilla.org/show_bug.cgi?id=1032371

**Comment by bobsilverberg** [comment]
This is good, @tessie, and works well.

I suggest you also update `test_users_who_completed_this_task` so that it verifies that the users are in the expected order.

**Comment by tessie** [comment]
@bobsilverberg .updated my pr

File: oneanddone/tasks/tests/test_models.py [hunk 1]
```diff
@@ -499,7 +499,7 @@ def test_abandoned_user_count(self):
     def test_all_attempts(self):
         eq_(len(self.task1.all_attempts), 11)
 
-    def test_incomplete_attempts(self):
```
Comment (tessie) [inline oneanddone/tasks/tests/test_models.py]: found two methods with same name.at L516

**Comment by bobsilverberg** [comment]
Thanks @tessie, this looks good. I took the liberty of squashing your commits for you, and then merging them, so this landed in https://github.com/mozilla/oneanddone/commit/8e7282162ad30a171cbaf1209ad36c331c8a322c

**Pull request closed without merging**